windows_service/datadog_checks/windows_service/windows_service.py

```diff
@@ -34,7 +38,7 @@ def check(self, instance):
 
         instance_hash = hash_mutable(instance)
         instance_key = self._get_instance_key(host, self.NAMESPACE, self.CLASS, instance_hash)
-        tags = [] if (host == "localhost" or host == ".") else [u'host:{0}'.format(host)]
+        tags = [] if (host == "localhost" or host == ".") else [u'host:{}'.format(host)]
```

Comment:
Why did you remove `0`? It is safer this way.

Reply:
We try to avoid this style because it's verbose. If there are more than 3 args then keyword args may be used.